File: Array/Trailing Zeros in Factorial.go
package main

import "fmt"

func main() {

	A := 4617
	result := trailingZeroes(A)
	fmt.Println(result)
}

func trailingZeroes(A int) int {
	ans, i, div := 0, 5, A/5

	for div > 0 {
		ans += div
		i *= 5
		div = A / i
	}

	return ans
}

File: Pythons Solutions/Trailing Zeros in Factorial.py
import re


class Solution:
    def trailingZeros(self, num):
        ans, i, div = 0, 5, num/5

        while div > 0:
            ans += div
            i *= 5
            div = num/i
        
        return int(ans)

obj = Solution()
print(obj.trailingZeros(25))